utbot-framework/src/main/kotlin/org/utbot/framework/codegen/model/visitor/CgAbstractRenderer.kt

```diff
@@ -98,6 +110,22 @@ internal abstract class CgAbstractRenderer(val context: CgContext, val printer:
 
     protected abstract val langPackage: String
 
+    //We may render array elements in initializer in one line or in separate lines:
+    //items count in one line depends on the value type.
+    protected fun arrayElementsInLine(constModel: UtModel): Int {
+        if (constModel is UtNullModel) return 10
+        return when (constModel.classId) {
+            intClassId,
+            byteClassId,
+            longClassId,
+            charClassId -> 8
+            shortClassId,
+            doubleClassId,
+            floatClassId -> 6
+            else -> 5
```

Comment:
It looks a little weird. Could I make a suggestion?
```suggestion
        return when (constModel.classId) {
            intClassId, byteClassId, longClassId, charClassId -> 8
            shortClassId, doubleClassId, floatClassId -> 6
            else -> 5
```

Also, is it true that only acceptable value in the `else` branch is `booleanClassId`? In this case, I suggest specifying it explicitly and throwing an error in the `else` branch.